tests/sdk/test_add.py

```diff
@@ -93,27 +94,37 @@ def test_add_composite_algo(client, algo_query, mocker):
     assert m_get.is_called()
 
 
-def test_add_data_sample(client, data_sample_query, mocker):
+def test_add_data_sample(client, data_sample_query, data_samples_query, mocker):
```

Comment:
When writing unit tests we should try to follow the rule: a test should check one thing or test one feature/a single case. 

Tests are then easier to read (we can quickly understand what's the goal of test) and it's then easier to understand what is the issue when a test is broken.

So you should not extand or group existing tests but create new ones.

Reply:
Maybe I can check the existing tests and apply this rule?

Reply:
That could be done later but it's not a priority right now and it's better to have a small scope in each PR. So in my opinion, this PR should only add a few missing tests and not change the existing ones.

Reply:
Okay!